src/components/ResourceBrowser/ResourceList/Sidebar.tsx

```diff
@@ -272,10 +272,29 @@ const Sidebar = ({
     }
 
     const handleInputChange = (newValue: string, actionMeta: InputActionMeta): void => {
-        if (actionMeta.action === 'input-change') {
-            setSearchText(newValue)
-            setMenuOpen(!!newValue)
+        if (actionMeta.action !== 'input-change') {
```

Comment:
Check can have a constant if possible

Reply:
TODO: will be incorporated from the feat/user-status-p3 branch